### PR TITLE
Add support for Python 3.12

### PR DESCRIPTION
**Pull request by pbauer**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,53 +7,58 @@
 from setuptools import find_packages
 
 install_requires = [
-    'setuptools',
-    'six',
-    'Jinja2',
+    "setuptools",
+    "six",
```
Comment (mauritsvanrees) [inline setup.py:11]: I am not behind my laptop, so I cannot check: is six still needed? It should not be.

File: setup.py [hunk 2]
```diff
@@ -7,53 +7,58 @@
...
+    version="1.0.1.dev0",
+    description="Bob renders directory structure templates",
+    long_description=read("README.rst") + "\n" + read("HISTORY.rst"),
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: Implementation :: CPython",
+        "Programming Language :: Python :: Implementation :: PyPy",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
```
Comment (mauritsvanrees) [inline setup.py:33]: I would be fine with dropping the EOL 3.7. But this is optional.

**Review by mauritsvanrees — APPROVED** [review]
LGTM. Two remarks, but fine to merge without changes.

File: setup.py (hunk 1 above)
Comment (pbauer) [reply setup.py:11]: I now removed six as well

**Pull request merged**